SOSE_tracer_budgets.ipynb

```diff
@@ -24,8 +24,11 @@
    "outputs": [],
    "source": [
     "from dask.distributed import Client, progress\n",
-    "from dask_kubernetes import KubeCluster\n",
-    "cluster = KubeCluster(n_workers=30)\n",
+    "from dask_gateway import Gateway\n",
+    "\n",
+    "gateway = Gateway()\n",
+    "cluster = gateway.new_cluster()\n",
+    "cluster.scale(30)\n",
```

Comment:
optional: switch all our uses of `cluster.scale` to `cluster.adapt`.

Reply:
Thought about it, but probably would need more testing.

Reply:
Sounds like we should wait to do this then.